x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -148,46 +193,35 @@ export const signalRulesAlertType = ({
             pageSize: searchAfterSize,
             tags,
           });
+        }
 
-          if (bulkIndexResult) {
-            logger.debug(
-              `Finished signal rule name: "${name}", id: "${alertId}", rule_id: "${ruleId}"`
-            );
-            await writeCurrentStatusSucceeded({
-              services,
-              currentStatusSavedObject,
-            });
-          } else {
-            await writeSignalRuleExceptionToSavedObject({
-              name,
-              alertId,
-              currentStatusSavedObject,
-              logger,
-              message: `Bulk Indexing signals failed. Check logs for further details \nRule name: "${name}"\nid: "${alertId}"\nrule_id: "${ruleId}"\n`,
-              services,
-              ruleStatusSavedObjects,
-              ruleId: ruleId ?? '(unknown rule id)',
-            });
-          }
-        } catch (err) {
+        if (bulkIndexResult) {
+          logger.debug(
+            `Finished signal rule name: "${name}", id: "${alertId}", rule_id: "${ruleId}"`
+          );
+          await writeCurrentStatusSucceeded({
+            services,
+            currentStatusSavedObject,
+          });
+        } else {
           await writeSignalRuleExceptionToSavedObject({
             name,
             alertId,
             currentStatusSavedObject,
             logger,
-            message: err?.message ?? '(no error message given)',
+            message: `Bulk Indexing signals failed. Check logs for further details \nRule name: "${name}"\nid: "${alertId}"\nrule_id: "${ruleId}"\n`,
```

Comment:
This is missing the `outputIndex` and really even though we started adding new lines in some cases to the log messages I think that is a bad practice we should discontinue as when log messages are "grepped" or being consumed by say our own product or other products they're easier to go from unstructured to structured when the new lines are removed.

Also, it just makes it easier for people to past a log line and not accidentally miss the line below or think they have multiple errors when they really have one line of an error. Something like this:

```ts
message: `Bulk Indexing signals failed. Check logs for further details Rule name: "${name}" id: "${alertId}" rule_id: "${ruleId}" output_index: "${outputIndex}"`,
```